full/Dockerfile

```diff
@@ -147,10 +147,22 @@ RUN curl -sSL https://rvm.io/mpapis.asc | gpg --import - \
         && gem install bundler --no-document"
 
 ### Rust ###
+USER root
```

Comment:
Nit: Please use `sudo` below instead of repeating `USER root` / `USER gitpod`.

Reply:
I personally prefer the `USER` but agree that repeated switches can be confusing in longer Dockerfiles.